specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
+          },
+          "supercededUpdateIds": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Superseded Update Ids"
+          },
+          "isSupercededUpdate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Is a superceded update"
+          },
+          "deploymentAction": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Deployment action"
+          },
+          "kbid": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Knowledge Base ID of a Windows Update"
+          },
+          "kbRevisionNumber": {
+            "type": "string",
+            "readOnly": true,
+            "description": "KB revision number"
+          },
+          "publishedDate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Reported published date for this update"
+          },
+          "productName": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product name"
+          },
+          "productVersion": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product version"
+          },
+          "productArch": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Product architecture"
+          },
+          "title": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Update title"
+          },
+          "classification": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Normalized update classification"
+          },
+          "packageClassification": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Linux package classification"
+          },
+          "packageSeverity": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Linux package severity"
+          },
+          "msrcSeverity": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Microsoft Security Response Center severity for Windows updates"
+          },
+          "externalURL": {
+            "type": "string",
+            "readOnly": true,
+            "description": "External documentation URL"
+          },
+          "msrcBulletinId": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Microsoft Security Response Center bulletin id"
+          },
+          "cveNumbers": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Common Vulnerabilities and Exposures numbers"
+          },
+          "rebootBehavior": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Reboot behavior"
+          },
+          "optional": {
```

Comment:
did you intend this to be a bool? string enum works better.

Reply:
OS specific, might be empty for certain cases. Keeping it string.